app/workers/reactions/update_relevant_scores_worker.rb

```diff
@@ -9,7 +9,11 @@ def perform(reaction_id)
       return unless reaction&.reactable
 
       reaction.reactable.touch_by_reaction if reaction.reactable.respond_to?(:touch_by_reaction)
-      reaction.reactable.sync_reactions_count if rand(6) == 1 && reaction.reactable.respond_to?(:sync_reactions_count)
```

Comment:
There actually don't seem to be too many places left where we use these randomized calls, but here is one of them. The main problem is that I completely guessed how long we throttle for (please let me know if you'd prefer a value other than 15 minutes), whereas the `rand` based approach just says "do this roughly one out of 6 times".

Reply:
I'm quite sure these numbers are based on observation of past performance woes in production so unless we literally benchmark this in prod and compare the before and after, we can rely on our collective judgments for the new intervals

Reply:
Knowing when to bust a cache is always hard and I have found that usually, you have to make your best guess at what will work and then be willing to tweak as necessary. 

That aside, at least we have something concrete to tweak and not a `rand` 🙈